runtime/doc/lsp.txt

```diff
@@ -689,6 +710,17 @@ declaration()                                      *vim.lsp.buf.declaration()*
 definition()                                        *vim.lsp.buf.definition()*
                 TODO: Documentation
 
+document_highlight()                        *vim.lsp.buf.document_highlight()*
+                Send request to server to resolve document highlights for the 
+                current text document position. This request can be associated
+                to key mapping or to events such as `CursorHold`, eg:
```

Comment:
these docs are generated from docstrings. Don't add it here directly, add it as a docstring in `lsp/buf.lua`. See `:help dev-lua-doc`

Reply:
Thanks, will do